entry.sh

```diff
@@ -13,7 +13,9 @@ if [ ! -e ${SENTINEL} ]; then
     exit 2
 fi
 
-echo "Rebuilding configure script"
-cd $SRC
-autoreconf -ivf -Werror $@
-echo "Done"
+if [ "$#" = "0" ]; then
+    echo "Rebuilding configure script using $(autoconf --version | head -n 1)"
+    exec gosu autoreconf -ivf -Werror $@
+fi
+
+exec "$@"
```

Comment:
I think it can be nice to be able to run with arbitrary commands. For example:

```console
$ docker run --rm -v$PWD:/src ghcr.io/corona10/cpython_autoconf:272-final autoconf --version
```

Reply:
Also, `gosu` is perhaps overkill, but you never know. We may have to install it explicitly, though.